Initial Documentation for Snap Sync (#552)
subspace/subspace-docs: docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx

- **[Polkadot.js Block Explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc-0.gemini-3h.subspace.network%2Fws#/explorer)**: For users familiar with the Polkadot ecosystem, this explorer offers a seamless experience for exploring the Subspace Network using the Polkadot.js interface.

### Snap Sync

> As of the June 11, 2024 version, a new option called Snap Sync is available for fast syncing. It only works for the initial sync, but many people report being fully synced within hours instead of days. The parameter is `--sync snap`. Currently, Snap Sync only works with a farming node, not an operator node.

Reviewer: We can slightly rephrase it, unless there're precedents where snap sync took longer than a few hours. 

```suggestion
As of the June 11, 2024 version, a new option called Snap Sync is available for fast syncing. It only works for the initial sync, but allow your node to be fully synced within hours instead of days. The parameter is `--sync snap`. Currently, Snap Sync only works with a farming node, not an operator node.
```
